diagnostic.md

```diff
@@ -44,7 +54,8 @@ weather === "sunny"
 ```
 
 What do these expressions evaluate to?  Explain your answers.
-
+weather = "sunny"  // weather will have a string value of 'sunny'
+weather === "sunny"  // this will compare to see if weather is 'sunny' and will evaluate to true.
```

Comment:
You got these two right but you left out the first part.
`let weather` will evaluate to undefined but I could see why you left this line out.